dbc2val/createvcan.sh

```diff
@@ -23,9 +23,8 @@
 # for the elm2canbridge
 #
 
-
-#Default dev, can be overriden by commandline
-DEV=elmcan
+#Default dev, can be overridden by commandline
+DEV=vcan0
```

Comment:
I think this breaks elmcanbridge as it calls this without parameter and expects an elmcan device

Reply:
we can change it back to elmcan

Reply:
Jupp, that would be good. Alternatively we can modify elmcan Python code to call it with `elmcan`